main_test.go

```diff
@@ -17,63 +17,117 @@ package main
 import (
 	"context"
 	"encoding/json"
-	"math/rand"
 	"net"
 	"net/http"
 	"net/http/httptest"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
-	"github.com/google/uuid"
 )
 
 func TestGenerate(t *testing.T) {
-	uuid.SetRand(rand.New(rand.NewSource(1)))
-	in := configInput{
-		xdsServerUri:     "example.com:443",
-		gcpProjectNumber: 123456789012345,
-		vpcNetworkName:   "thedefault",
-		ip:               "10.9.8.7",
-		zone:             "uscentral-5",
-	}
-	want := `{
```

Comment:
I want a test that checks the literal byte output, like was done previously. Otherwise there is no verification for the JSON key names and such. I don't actually see much value of using the parsed version instead of the string, as it is actually more verbose.

Reply:
Done.